packages/core/src/components/fields/SchemaField.js

```diff
@@ -71,17 +71,41 @@ function Label(props) {
   );
 }
 
-function LabelInput(props) {
-  const { id, label, onChange } = props;
-  return (
-    <input
-      className="form-control"
-      type="text"
-      id={id}
-      onBlur={event => onChange(event.target.value)}
-      defaultValue={label}
-    />
-  );
+class LabelInput extends React.Component {
+  constructor(props) {
+    super();
+    this.state = {
+      value: props.label,
+    };
```

Comment:
I'm curious why you switched to a full React compnent when you could have just used the `useState()` hook for this? There is a lot more overhead for full components vs a functional component

Reply:
@heath-freenome Better consistency as all components in the same directory use class when using `this.state` and there is no `useState` hook. I agree with you and we might want to refactor :) For this PR, it'd be better to keep the scope small.

Reply:
I'm not sure what you mean by "there is not `useState` hook"... See the documentation: https://reactjs.org/docs/hooks-state.html

Reply:
@heath-freenome I forgot to say `in the same directory`. I meant the below:
> $ grep -r useState . react-jsonschema-form/packages/core/src/components
> # Not found